hasher-matcher-actioner/scripts/submit_content_test.py

```diff
@@ -0,0 +1,165 @@
+#! /usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import os
+import json
+import base64
+import requests
+import dataclasses
+import typing as t
+from urllib.parse import urljoin
+from script_utils import HasherMatcherActionerAPI
+
+from hmalib.common.evaluator_models import Action, ActionRule
+from hmalib.common.classification_models import Label, ActionLabel, ClassificationLabel
+from hmalib.common.actioner_models import ActionPerformer, WebhookGetActionPerformer
+
+
+class DeployedInstanceTestHelper:
+    """
+    Class around testing a deployed instance of HMA from Content Submission to Hash - Match - Action
+    by checking that the expected values are found
+    """
+
+    SAMPLE_PG_ID = "inria-holidays-test"
+
+    def __init__(
+        self,
+        api_url: str,
+        api_token: str = "",
+        client_id: str = None,
+        refresh_token: str = None,
+    ) -> None:
+        if not api_token and (not client_id or not refresh_token):
+            print(
+                "Test requires an api_token OR a client_id + refresh_token to function"
+            )
+            exit(1)
```

Comment:
nit: Calling `exit()` from a class method can lead to hard-to-debug issues. Instead raise an exception.

I understand that this is not production code, so am fine without this being addressed.